File: routes/typescript-to-rust/certification/artifacts/development/route_harness.rs
include!("migrated.rs");

fn main() {
    let actual_0 = calculate(100.0, 20.0);
    assert!(actual_0 == 120.0, "case 0");
    println!("ELMOS_OBSERVATION\t0\trust-debug\t{:?}", actual_0);
    let actual_1 = calculate(-1.0, 5.0);
    assert!(actual_1 == 0.0, "case 1");
    println!("ELMOS_OBSERVATION\t1\trust-debug\t{:?}", actual_1);
    let actual_2 = calculate(7.0, -2.0);
    assert!(actual_2 == 5.0, "case 2");
    println!("ELMOS_OBSERVATION\t2\trust-debug\t{:?}", actual_2);
}

File: routes/csharp-to-python/certification/artifacts/development/route_harness.py
import json
import migrated
actual_0 = migrated.calculate(100, 20)
assert actual_0 == 120
print("ELMOS_OBSERVATION\tjson\t" + json.dumps({"case_id": 0, "value": actual_0}, sort_keys=True, separators=(",", ":")))
actual_1 = migrated.calculate(-1, 5)
assert actual_1 == 0
print("ELMOS_OBSERVATION\tjson\t" + json.dumps({"case_id": 1, "value": actual_1}, sort_keys=True, separators=(",", ":")))
actual_2 = migrated.calculate(7, -2)
assert actual_2 == 5
print("ELMOS_OBSERVATION\tjson\t" + json.dumps({"case_id": 2, "value": actual_2}, sort_keys=True, separators=(",", ":")))
